src/components/Stepper/Stepper.tsx

```diff
@@ -1,14 +1,18 @@
 import { type ReactElement } from "react";
 
-const Stepper = () => {
+type StepperProps = {
+  step: number;
+};
+
+const Stepper = ({ step }: StepperProps) => {
   const renderSteps = (): ReactElement[] => {
     const steps: number = 4;
-    const activeStep: number = 0;
+    const activeStep: number = step;
 
     return Array.from({ length: steps }, (_, i) => (
       <span
         key={i}
-        className={`step ${i === activeStep ? "step--active" : ""}`}
+        className={`step ${i <= activeStep ? "step--active" : ""}`}
```

Comment:
if the condition is not met you will end up with className "step" is this corrent?